### ISSUE TITLE
Skin QueueStatusFrame

### ISSUE THREAD

**Issue by DaleHuntGB** [frozn/TipTac#97]
Hi there,

Any chance you can skin the QueueStatusFrame to match the Tooltip?

This was the information I could find.

![QueueStatusFrame](https://user-images.githubusercontent.com/95256492/190932987-ff614352-27b0-44db-908a-8fbc21341449.png)

**Comment by frozn** [comment]
Good idea. I added the styling of QueueStatusFrame. Here's the dev version:

[TipTac-dev.zip](https://github.com/frozn/TipTac/files/9601979/TipTac-dev.zip)

**Comment by DaleHuntGB** [comment]
Just tested it now, seems to be working pretty well. 

Will report back anything that I can.
